impeller/renderer/backend/vulkan/context_vk.cc

```diff
@@ -572,4 +574,45 @@ std::shared_ptr<CommandQueue> ContextVK::GetCommandQueue() const {
   return command_queue_vk_;
 }
 
+// Creating a render pass is observed to take an additional 6ms on a Pixel 7
+// device as the driver will lazily bootstrap and compile shaders to do so.
+// The render pass does not need to be begun or executed.
+void ContextVK::InitializeCommonlyUsedShadersIfNeeded() const {
```

Comment:
Its really unclear how this is about shaders. Perhaps `InitializeCommonlyUsedRenderPassesIfNeeded`?

Reply:
The doc comment describes why this is about shaders

Reply:
I'm landing this but we can have another follow up discussion